config/package/conformance.yml

```
@@ -0,0 +1,7 @@
+packages:
+- name: quay
+  package: quay.io/crossplane/test:v0.1.0 
+- name: google artifact registry
+  package: gcr.io/crossplane/test:v0.1.0
```

This will be the image we want to use :)
```suggestion
  package: us-west1-docker.pkg.dev/crossplane-playground/xp-install-test/configuration:main
```

---

Ohkay, actually they were just the placeholder registries and packages that were added to be replaced by the ones we are actually gonna use once I'd been provided with the packages. Do I remove the quay package or we will be having a copy of that packages pushed to quay and other supported registries?

---

We can just start with GCR and expand to others after we have the framework in place 👍🏻